ecommerce/extensions/payment/views/payu.py

```diff
@@ -188,10 +188,8 @@ def get(self, request, *args, **kwargs):
         except:  # pylint: disable=bare-except
             return redirect(reverse('payment_error'))
 
-        receipt_url = get_receipt_page_url(
-            order_number=basket.order_number,
-            site_configuration=basket.site.siteconfiguration
-        )
+        # Redirect to a static page as per Campus request (ticket #12141)
+        receipt_url = 'https://www.campusromero.pe/compra-exitosa/'
```

Comment:
Can this be taken from the settings?

something like
settings.PAYU_RECEIPT_URL,  by default it can be an empty string.  We can override it with /edx/etc/ecommerce.yml and then use
```python
receipt_url = get_receipt_page_url(
             site_configuration=basket.site.siteconfiguration,
             receipt_url = receipt_url_taken_from_settings
    )
```

We are not passing any params, (order_number) to the static page? is that ok? I guess it is

Reply:
Yes, and I just saw you can pass a override_url parameter to get_receipt_page_url. So it would be:

Reply:
Yes, but I'd still need to change that part of the code for something like `receipt_url = get_receipt_page_url(
             site_configuration=basket.site.siteconfiguration,
             receipt_url = receipt_url_taken_from_settings,
             override_url = settings.PAYU_RECEIPT_URL
    )` There is no problem with not passing params to the static page. Those params where only used to display the order details in the default receipt page.

Reply:
I like your suggestion though, I'll work on it.

Reply:
I am sorry, I made a typo:

it should be with override_url instead of receipt_url as parameter name.